packages/dapp/src/components/QRCodeScanner/index.tsx

```diff
@@ -24,6 +24,7 @@ const QRCodeScanner = ({
   const onScanFailure = (_: any) => {};
 
   useEffect(() => {
+    console.log('setting scanner..');
```

Comment:
Maybe remove most of the debug comments and only leave the most important ones (for errors logs) ?